### PR TITLE
WIP: Ize 588 typeit react library throws a hook error

### PR DESCRIPTION
**Pull request by Lixxies**

### CODE REVIEW

File: website/components/typeItAnimation.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import TypeIt from "typeit-react"
+
+export default function TypeItAnimation() {
+    return (
+        <div>
+             <TypeIt>nfddfdfdfdfdffd</TypeIt>
+        </div>
+    )
+}
```
Comment (AutomationD) [inline website/components/typeItAnimation.js:9]: Let's add new line in the end of the file

File: website/components/typeItAnimation.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import TypeIt from "typeit-react"
+
+export default function TypeItAnimation() {
+    return (
+        <div>
+             <TypeIt>nfddfdfdfdfdffd</TypeIt>
```
Comment (AutomationD) [inline website/components/typeItAnimation.js:6]: Is this a correct content? 🤔

**Review by AutomationD — APPROVED** [review]
Looks good, but please address the minor comments.

File: website/components/typeItAnimation.js (hunk 1 above)
Comment (Lixxies) [reply website/components/typeItAnimation.js:9]: done

File: website/components/typeItAnimation.js (hunk 2 above)
Comment (Lixxies) [reply website/components/typeItAnimation.js:6]: Im gonna finish this component today

Comment (AutomationD) [reply website/components/typeItAnimation.js:6]: Cool. It's your call then to merge this PR.

**Pull request merged**